docker/prepare_proxy_test.sh

```diff
@@ -1,16 +1,57 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
IMHO using the golden-file style test ( from the new repo ) is sufficient - we want to verify the script sets the iptables as expected. 

I'm not sure we need to be testing that iptables is implemented correctly, but if we do it can be also done on a single (local) docker image, by checking the capture counters. Starting 3 docker images in a k8s cluster seems overkill and overcomplex.